src/SearchBar.js

```diff
@@ -0,0 +1,13 @@
+import React, { useContext } from "react";
+import {searchContext} from "./search/searchContext";
+
+export default function SearchBar() {
+  const onChange = (e) => setSearchBarValue(e.target.value || null);
```

Comment:
`const onChange = (e) => setSearchBarValue(e.target.value || null);` why null? Just simply set to whatever it is empty string or not, no need for the null.